Packs/DatadogCloudSIEM/Integrations/DatadogCloudSIEM/DatadogCloudSIEM.py

```diff
@@ -0,0 +1,791 @@
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=INTEGRATION_CONTEXT_NAME,
+        outputs_key_field="Tag",
+        outputs=context_output if tags else [],
+    )
+
+
+def add_tags_to_host_command(
+    configuration: Configuration, args: Dict[str, Any]
+) -> Union[CommandResults, DemistoException]:
+    """
+     This function adds tags to a specified host in Datadog.
+
+     Args:
+     configuration (Configuration): The configuration object for Datadog.
+     args (Dict[str, Any]): A dictionary containing the following keys:
+     - host_name (str): The name of the host to add tags to.
+     - tags (str or List[str]): The tags to add to the host, separated by commas or provided as a list.
+
+    Returns:
+     CommandResults: The object containing the command results, including the readable output, outputs prefix,
+      outputs key field, and outputs data.
+    """
+    host_name = args.get("host_name")
+    tags = argToList(args.get("tags"), ",")
+
+    body = HostTags(host=host_name, tags=tags)
+
+    with ApiClient(configuration) as api_client:
+        tags_api = TagsApi(api_client)
+        response = tags_api.create_host_tags(host_name=host_name, body=body)
+    if response and response.get("host"):
+        readable_output = "Tags added to host successfully!"
+        output_context = {"Tag": response.get("tags"), "HostTag": response.get("host")}
+    else:
+        readable_output = ERROR_MSG
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=INTEGRATION_CONTEXT_NAME,
+        outputs_key_field="Tag",
+        outputs=output_context if response else [],
+    )
+
+
+def update_host_tags_command(
+    configuration: Configuration, args: Dict[str, Any]
+) -> Union[CommandResults, DemistoException]:
+    """
+    This function updates the tags of a specified host in Datadog.
+
+    Args:
+     configuration (Configuration): The configuration object for Datadog.
+     args (Dict[str, Any]): A dictionary containing the following keys:
+    - host_name (str): The name of the host to update tags for.
+    - tags (str or List[str]): The new tags to set for the host, separated by commas or provided as a list.
+
+    Returns:
+    CommandResults: The object containing the command results, including the readable output, outputs prefix,
+     outputs key field, and outputs data.
+
+    """
+    host_name = args.get("host_name")
+    tags = argToList(args.get("tags"), ",")
+
+    body = HostTags(host=host_name, tags=tags)
+
+    with ApiClient(configuration) as api_client:
+        tags_api = TagsApi(api_client)
+        response = tags_api.update_host_tags(host_name=host_name, body=body)
+
+    if response and response.get("host"):
+        output_context = {"Tag": response.get("tags"), "HostTag": response.get("host")}
+        readable_output = "Tags updated to host successfully!"
+    else:
+        readable_output = ERROR_MSG
```

Comment:
Can you give a more elaborate error than something went wrong ? can we get what went wrong from the API ?